### DISCUSSION TITLE
Where is token stored?

### DISCUSSION THREAD

**Discussion by oriiyx** [lexik/LexikJWTAuthenticationBundle#1126] [General]
Hej :)

I implemented and went through your bundle - what I could not figure out is; Where do you store the created tokens on the system?

I didn't see any migrations - I checked and scrumbed over the $_SYSTEM with XDebug but didn't find anything.
I went through documentations - here and on symfony page - I could've done a shoddy job reading but nothing jumped at me so that I could understand clearly.
The code - I tried for 10 mins... when I figured I could just ask you guys 🤷 

Hope I don't break any rules with asking these things...

Cheers guys :)

**Comment by chalasr** [comment]
Hi, unless you explicitly configure it to make it [set cookies](https://symfony.com/bundles/LexikJWTAuthenticationBundle/current/1-configuration-reference.html#automatically-generating-cookies), tokens are not stored anywhere: once delivered by the server as par of the body of a successful authentication response, the client has to store it on its own and pass it to the server through all subsequent requests that require authentication. Hope it's clear :)
